local_sales/.env.example

```diff
@@ -1,4 +0,0 @@
-DB_NAME=xxxxxx
```

Comment:
Why are you removing this file? 
If you need to remove it you can do it in another PR, to maintain only the `offers crud` changes in this PR